fdm-app/app/routes/farm.$b_id_farm.$calendar.rotation.harvest._index.tsx

```
@@ -0,0 +1,1004 @@
+
+                return {
+                    ...field,
+                    hasHarvest,
+                    harvestApplication,
+                    harvestableAnalysis,
+                }
+            }),
+        )
+
+        let firstFieldWithData
+        if (targetCultivation.b_lu_harvestable === "once") {
```

Biome (2.1.2)

[error] This variable implicitly has the any type.

Variable declarations without type annotation and initialization implicitly have the any type. Declare a type or initialize the variable with some value.

(lint/suspicious/noImplicitAnyLet)